ide-eclipse.html

```diff
@@ -0,0 +1,366 @@
+<h2>JavaFX <span class="JFX">11</span> and Eclipse</h2>
+
+<p>This section explains how to use Java <span class="JDK">11</span> and JavaFX <span class="JFX">11</span> from Eclipse. 
+    Version Eclipse 2018-09 (4.9) Build id: I20180906-0745 was used for the following screenshots.
+</p>
+
+<p>
+    Download an appropriate <a href="https://jdk.java.net/11/">JDK <span class="JDK">11</span></a> for your operating system. 
+    Make sure <kbd>JAVA_HOME</kbd> is properly set to the Java <span class="JDK">11</span> installation directory. 
+</p>
+
+<p>
+    Include the new JDK as <kbd>Installed JREs</kbd> in <kbd>Eclipse -> Preferences -> Java -> Installed JREs -> Add</kbd>.
+</p>
+
+<div class="alert alert-warning">
+    <strong>Note: </strong>
+    If you get this message when you add JDK <span class="JDK">11</span>
+
+<pre><code>
+You selected a JRE that this version of Eclipse JDT does not yet support fully. Some of the features may not work as expected.
```

Comment:
We might want to change the style here. The entire code block inside a warning div doesn't look very nice.

Reply:
Here is a sample without and with background color:

![capture](https://user-images.githubusercontent.com/3197675/48250107-2fb24280-e423-11e8-86c4-147ca44ef7f4.PNG)

Reply:
Agree